dogfood.md

```diff
@@ -0,0 +1,167 @@
+# MemeoList dog food app
+Previously while working on the mobile team, Summers (GitHub user secondsun) wrote a simple application "memeolist" (a portmanteau of memo-list and meme).  This exercise let him explore how the RHMAP 3.x product worked if he were building an app that was very similar to other apps he had seen. He then compiled his experiences into a post mortem, fixed a few annoying bugs, and gained a deeper understanding of how RHMAP worked.
+
+Finally he shared a running instance with the mobile-team so that we could get a "stress test" of the application.  Overall this was a fun and successful project for us and we should replicate that exercise with Aerogear so we can test our assumptions and stretch our knowledge.
+
+A video demonstration for the project is here : https://drive.google.com/file/d/0B64fta5NYF0TUmNFZy0zQ3JTRjQ/view
+The source code for the original versions are here : https://github.com/secondsun/memeolist-android and https://github.com/secondsun/art-henry-cloud-app/
+
+## Project Description
+
+Memeolist is a social media application for the AeroGear community where they can share funny images (memes) with each other.  The application will consist of AeroGear services : UPS, KeyCloak, and Metrics; a GraphQL API layer fronted by a custom connector, and client applications using out Xamarin, iOS, Android, and JavaScript SDKs.
+
+## Core Features / User Stories
+This section of the proposal describes the actions that a user can perform in the application and provides some basic constraints.
+
+### A User Logs in
+Users should be able to log in with a social login that is provided by the underlying operating system. For example, On Android these accounts will be exposed through the account picker. On platforms where this isn't provided, we should fall back to an OAuth flow using the system browser.
+
+### Users can manage a self profile
+Users should be able to upload and edit information about themselves in the application.  This will include email, a display name, a profile picture, and a short biography.  As much of this information should be auto filled from the social login when a user first logs in.
```

Comment:
@secondsun If we're planning on publishing the Memolist App to the various app stores, are there any GDPR concerns here?

Reply:
There are almost certainly GDPR concerns.  

If it were just you or I running the app as long as we didn't sell the data, were up front with what we used the data for, let people download their data, and let people delete their data we would be ok.  If a large open source company is hosting/storing the data then they have extra responsibilities.